scripts/trello/cards.js

```diff
@@ -0,0 +1,7 @@
+async function postComment(cardId, text, { trelloApiKey, trelloOAuth1 } = {}) {
+    return post(
+        `cards/${cardId}/actions/comments`,
+        `text=${text}`,
```

Comment:
@AdriAt360 Could be more maintainable to pass `{ text }` in here and let your client lib do the encoding

Reply:
@pp0rtal yes sir! https://github.com/360Learning/360learning.github.io/pull/10/commits/0ed2235e2f538ed602527435750b7130cc312015